lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,33 @@
+[GitLab Generic Packages Repository](https://docs.gitlab.com/ee/user/packages/generic_packages/) supports looking up package versions from all types of packages registry supported by GitLab and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
```

Comment:
Add a note that this requires GitLab >= 13.5

Reply:
The [package api](https://docs.gitlab.com/ee/api/packages.html) was introduced in Gitlab 11.8. It's a simple API that support listing packages from all packages registries supported by Gitlab (As of now: Generic, Composer, Conan, Debian, Go Proxy, Helm, Maven, npm, nuget, PyPi and Ruby Gems. Gitlab 13.5 introduced the support for the generic package registry, but this code will also function for all other type of packages.

For performance reason, It's better to use it with Gitlab >=12.9, because of the added "package_name" argument, that allow to only get info about the package we are looking (fuzzy search by lookupName).